src/NServiceBus.Persistence.AzureStorage.AcceptanceTests/Sagas/When_doing_request_response_between_sagas.cs

```diff
@@ -54,7 +54,7 @@ protected override void ConfigureHowToFindSaga(SagaPropertyMapper<RequestRespons
 
                 public class RequestResponseRequestingSagaData : ContainSagaData
                 {
-                    public virtual Guid CorrIdForResponse { get; set; } //wont be needed in the future
+                    public virtual Guid CorrIdForResponse { get; set; }
```

Comment:
:)
I read this case thoroughly and I can tell that the correlation id is not set. I can't see why this id should be deleted though. The setter is no longer used, but we need the correlation between request-response? Can you see what's in my blind spot?